recipe/meta.yaml

```diff
@@ -139,11 +152,6 @@ outputs:
         - make   # [unix]
         - llvmdev {{ version }}     # [build_platform != target_platform]
       host:
-        - {{ pin_subpackage("clang", exact=True) }}
-        - {{ pin_subpackage("clangxx", exact=True) }}
-        - {{ pin_subpackage("clang-tools", exact=True) }}
-        - {{ pin_subpackage("libclang", exact=True) }}
-        - {{ pin_subpackage("libclang-cpp", exact=True) }}
```

Comment:
This (and the other `host:` removals) will be a persistent clobber hazard. This should be a last resort, I'd like to at least discuss alternatives before merging this without review

Reply:
Huh? `files:` will just blindly copy stuff into the artefact, and to avoid clobbering clangdev with all the rest requires matching the exact set of paths that should be included. Too few, and clangdev is incomplete; too many, and clangdev clobbers some of its run dependencies. That's not a setup I'm particularly keen on

Reply:
You didn't respond to my quoted comment. I think the solver is messing up here, and rather than pessimizing the recipe, I'd be more interested in trying whether a translation to v1 does not run into the same issue. I'm willing to do the v1 translation, but I don't want to invest that time if you you are against it for some other reason. Could you please comment on that?

Reply:
Not really. This PR is the only way I found.